README.md

```diff
@@ -52,14 +51,14 @@ There are two main, recommended methods of using the module:
 
 ### With the CLI
 
-The easiest way to use it is with the CLI. In the directory where your
+The easiest way to use it is with the [webpack CLI](https://webpack.js.org/api/cli/). In the directory where your
 `webpack.config.js` is, run:
 
 ```console
-node_modules/.bin/webpack-dev-server
+node_modules/.bin/webpack serve
```

Comment:
`npx webpack serve`

Reply:
`npx` could use the globally installed module, which would lead to errors. I think this was to demonstrate one should use the local module. Binaries can be installed in other locations than `node_modules/.bin`, but those who do that would be advanced users.